README.md

```diff
@@ -60,11 +60,13 @@ func main() {
 Create a VPS
 
 ```go
+enableIP6 := false
+
 instanceOptions := &govultr.InstanceCreateReq{
   Label:                "awesome-go-app",
   Hostname:             "awesome-go.com",
-  Backups:              true,
-  EnableIPv6:           true,
+  Backups:              "false",
```

Comment:
Backups are a string because they are either "enabled" or "disabled" on the API

https://www.vultr.com/api/#operation/create-instance

Reply:
Ok, that makes sense and is as per the test case!

https://github.com/vultr/govultr/blob/019c4d29bf308a32888c6b93094925f69164c281/instance_test.go#L525-L526